src/keybindings.h

```diff
@@ -279,6 +279,8 @@ enum GeanyKeyBindingID
 												 * @since 1.38 (API 240) */
 	GEANY_KEYS_PROJECT_NEW_FROM_FOLDER,			/**< Keybinding.
 												 * @since 2.0 (API 243) */
+	GEANY_KEYS_TOGGLE_MENUBAR,					/**< Keybinding.
+												 * @since 2.1 (API TODO) */
```

Comment:
This needs updating (and the API version to match)
```suggestion
												 * @since 2.2 (API 251) */
```